### DISCUSSION TITLE
Slug update doesn't work

### DISCUSSION THREAD

**Discussion by kskrlinnorth2** [spatie/laravel-sluggable#243] [General]
According to documentation and this section https://github.com/spatie/laravel-sluggable#prevent-slug-updates slug updates are included by default. However that doesn't work at all. Also calling generateSlug() on model to recreate slug does nothing...

So how to update slugs?

This is my code:
```
use HasTranslations, HasTranslatableSlug;

public $translatable = [
    'name',
    'slug',
];

public function getSlugOptions(): SlugOptions
{
    return SlugOptions::create()
        ->generateSlugsFrom(['name', 'starts_at_iso'])
        ->saveSlugsTo('slug');
}
```

**Comment by kskrlinnorth2** [comment]
Looks like only slugs generated via callable are getting updated... which is never mentioned in docs... But this also updates slug for all languages, even those which have no name in DB...

```
public function getSlugOptions(): SlugOptions
{
    return SlugOptions::createWithLocales(['en', 'hr'])
        ->generateSlugsFrom(function ($model, $locale) {
            return "{$model->name} {$model->startsAtIso}";
        })
        ->saveSlugsTo('slug');
}
```

**Reply by digitcreators** [reply]
this code is not working in my case.
  ```
public function getSlugOptions() : SlugOptions
    {
        return SlugOptions::createWithLocales( config('translatable.locales') )
            ->generateSlugsFrom(function($model, $locale) {
                return "{$model->title} {$model->startsAtIso}";
            })
            ->saveSlugsTo('slug');
    }

``` 
am doing it like this, still not updating

**Reply by kskrlinnorth2** [reply]
My solution works for my case (when I update name or starts_at value I get new updated slug).
You can send your complete code for this model and your update method to check.
